jobs/broken-site-report-ml/broken_site_report_ml/main.py

```diff
@@ -0,0 +1,202 @@
+import click
+import datetime
+import logging
+import requests
+import time
+
+from google.cloud import bigquery
+
+BUGBUG_HTTP_SERVER = "https://bugbug.herokuapp.com"
+CLASSIFICATION_LABELS = {0: "valid", 1: "invalid"}
+
+
+def classification_http_request(url, reports):
+    reports_list = list(reports.values())
+    response = requests.post(
+        url, headers={"X-Api-Key": "docker-etl"}, json={"reports": reports_list}
+    )
+
+    response.raise_for_status()
+
+    return response.json()
+
+
+def get_reports_classification(model, reports, retry_count=21, retry_sleep=10):
```

Comment:
I've reused the same logic for retry/sleep as in https://github.com/mozilla/bugbot/blob/master/bugbot/bugbug_utils.py